desktop/libs/notebook/src/notebook/templates/editor2.mako

```diff
@@ -1290,10 +1290,6 @@ There is no bridge to KO for components using this integration. Example using in
 
   window.EDITOR_ENABLE_QUERY_SCHEDULING = '${ ENABLE_QUERY_SCHEDULING.get() }' === 'True';
 
-  window.EDITOR_ID = ${ editor_id or 'null' };
-
-  window.NOTEBOOKS_JSON = ${ notebooks_json | n,unicode };
-
```

Comment:
This change is about getting rid of these so that the parent script element can be removed in the following commit.